### PR TITLE
Test cleanup

### PR DESCRIPTION
**Pull request by brittcyr**

### CODE REVIEW

File: tests/utils/utils.ts [hunk 1]
```diff
@@ -0,0 +1,120 @@
+import { Provider, BN } from '@project-serum/anchor';
```
Comment (DonDuala) [inline tests/utils/utils.ts:1]: This is testing all these helper functions? Or targeted at creating SOs?

File: programs/staking-options/src/instructions/name_token.rs [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+pub use crate::*;
+
+pub fn name_token(
+    ctx: Context<NameToken>,
+    strike: u64
+) -> Result<()> {
+    // Verify the mint
+    check_mint!(ctx, strike, bump);
+
+    msg!("Calling into metaplex for {}", ctx.accounts.state.so_name);
```
Comment (DonDuala) [inline programs/staking-options/src/instructions/name_token.rs:12]: Comment with an example of what an expected name should be?

**Review by DonDuala — APPROVED** [review]

File: tests/utils/utils.ts (hunk 1 above)
Comment (brittcyr) [reply tests/utils/utils.ts:1]: this is the anchor test for SO

File: programs/staking-options/src/instructions/name_token.rs (hunk 2 above)
Comment (brittcyr) [reply programs/staking-options/src/instructions/name_token.rs:12]: didnt mean to include this. putting it in another change

**Pull request merged**